OpenRA.Mods.Common/Traits/BotModules/McvManagerBotModule.cs

```diff
@@ -63,7 +63,7 @@ public CPos GetRandomBaseCenter()
 
 		IBotPositionsUpdated[] notifyPositionsUpdated;
 		IBotRequestUnitProduction[] requestUnitProduction;
-
+		IResourceLayer resourceLayer;
 		CPos initialBaseCenter;
```

Comment:
I would recommend to move initialBaseCenter to player.cs and remove the duplicated code in MCVManager and BasebuilderModule.

Reply:
bot code is separated from the rest of the codebase by design, I don't think we should move it to the player class

Reply:
@phrohdoh have you an idea what we can do at this point?

Reply:
Why not keep track of the base centers inside the AI module instead of using the initial center from `Player.cs`?